test/config_test.exs

```diff
@@ -1,24 +1,33 @@
 defmodule PromisepayEx.ConfigTest do
   use ExUnit.Case
+  alias PromisepayEx.Config
 
   test "auth initialization global" do
     auth =
-      [ username: "username",
+      [
+        username: "username",
         token: "token",
         environment: "environment",
-        api_domain: "api_domain" ]
-    PromisepayEx.Config.set(auth)
+        api_domain: "api_domain",
```

Comment:
I always put an extra comma at the end of an array (Elixir List). Coding conventions :)